### DISCUSSION TITLE
Volume fraction of specific number of element sets?

### DISCUSSION THREAD

**Discussion by pouryajavanmard** [neperfepx/neper#535] [Q&A] [answered]
Hi

I am currently working on mesoscale model of concrete which is two-phase (aggregate and mortar) and I am using Neper to generate the meshing and grains. So, I command the tessellation of grains and meshing it after, then I convert it to abaqus to assign the material properties and boundary conditions. Then, I assigned the the element sets to the phases that I defined but I need the volume fraction of Aggregates.

My problem is how can I get the volume of each element sets?

I thought by counting the number of elements that each element sets have and then divide it by the whole number of elements I can get it but apparently it is not.

Thank you very much.

**Comment by rquey** [comment, marked as answer]
Use `-statelset vol`.
